### ISSUE TITLE
Newly Created Customers Don't Need Subscription To CV Search

### ISSUE THREAD

**Issue by Samyoul** [Samyoul/dywaj-issues#63]
Likely a simple logic issue. Customers should have a deal, to perform an action.

**Comment by Samyoul** [comment]
@Nyanko-sensei 

>Seems like permissions issue.
>
>In database (after seed) we have permission "search cvs" and on route we have ['prefix' => 'cv-search', 'middleware' => 'permission:search'] . So which permission should be on route <url>/cv-search/submit: "search" or "search cvs:? Also there are other routes, that have permission "search". After seeding there is no such permission, so it will always throw PermissionDoesNotExist exception (except if  user has that in their action bucket). Is this expected behavior?

**Comment by Samyoul** [comment]
>Is this expected behavior?

No, it isn't expected behaviour. The permission name is wrong. The permission should be `['prefix' => 'cv-search', 'middleware' => 'permission:search cvs']` . But we've changed the database value to match `search` permission.

I've just realised one of the other fixes I've applied fixes this problem. There is no problem here for the moment.

The middleware controls if a user should have access to the feature, dependant on the user's role(s). The action buckets control how much activity the user can control.

However I will create a new issue for the mismatch between the seed permission name and the route middleware permission name that you mention.
